src/main/java/net/barrage/school/java/ecatalog/app/ProductSource/FakeProductSource.java

```diff
@@ -1,4 +1,4 @@
-package net.barrage.school.java.ecatalog.app;
+package net.barrage.school.java.ecatalog.app.ProductSource;
```

Comment:
In Java we usually don't use CamelCase. Because Java is cross-platform and Win's file system ignores case.

Reply:
Okay, maybe its not because of windows filesystem. Probably it's just a tradition. But u can check yourself - nobody use CamelCase for package names!